packages/esm-patient-registration-app/src/patient-registration/field/person-attributes/person-attribute-field.test.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { usePersonAttributeType } from './person-attributes.resource';
+import { PersonAttributeField } from './person-attribute-field.component';
+import { useConceptAnswers } from '../field.resource';
+import { Form, Formik } from 'formik';
+import { FieldDefinition } from '../../../config-schema';
+
+jest.mock('./person-attributes.resource'); // Mock the usePersonAttributeType hook
+jest.mock('../field.resource'); // Mock the useConceptAnswers hook
+
+jest.mock('formik', () => ({
+  ...jest.requireActual('formik'),
+}));
+
+const mockedUsePersonAttributeType = usePersonAttributeType as jest.Mock;
+const mockedUseConceptAnswers = useConceptAnswers as jest.Mock;
+
+let fieldDefinition: FieldDefinition;
+
+describe('PersonAttributeField', () => {
+  let mockPersonAttributeType = {
+    format: 'java.lang.String',
+    display: 'Reffered by',
```

Comment:
```suggestion
    display: 'Referred by',
```
Update the spellings in other places as well

Reply:
done @Piumal1999